utils.rollup.config.js

```diff
@@ -0,0 +1,39 @@
+import typescript from 'rollup-plugin-typescript2';
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import json from '@rollup/plugin-json';
+import externals from 'rollup-plugin-node-externals';
+
+/**
```

Comment:
:question: usually we put the header on top of the file, is there a technical reason to not do this here?

Reply:
yes it was mistake auto added in wrong place